python/lsst/ts/observatory/control/auxtel/atcs.py

```diff
@@ -1288,16 +1298,21 @@ async def offset_xy(self, x, y, relative=False):
             `False` (default) offset replaces any existing offsets.
         """
 
-        self.log.debug(f"Applying x/y offset: {x}/ {y} ")
+        self.log.debug(f"Calculating x/y offset: {x}/ {y} ")
         azel = await self.rem.atmcs.tel_mount_AzEl_Encoders.aget()
-        nasmyth = await self.rem.atmcs.tel_mount_Nasmyth_Encoders.aget()
+        self.log.debug("got azel event ")
+        await asyncio.sleep(0)
+        nasmyth = await self.rem.atmcs.tel_mount_Nasmyth_Encoders.aget(timeout=1)
+        self.log.debug("got nasmyth event ")
+        await asyncio.sleep(0)
         angle = (
             np.mean(azel.elevationCalculatedAngle)
             - np.mean(nasmyth.nasmyth2CalculatedAngle)
             + 90.0
         )
         el, az, _ = np.matmul([x, y, 0.0], self.rotation_matrix(angle))
-
+        await asyncio.sleep(0)
```

Comment:
it is calling an `await` just after this sleep, I don't think it is necessary to add this sleep here.